module/modmanager/manager.go

```diff
@@ -203,6 +203,21 @@ func (mgr *Manager) Handles() map[string]modlib.HandlerMap {
 	return res
 }
 
+// / this function
+func whiteListViamModules(confs ...config.Module) (
```

Comment:
[nit] Could we say `allowlist` instead of `whitelist`? I think the former is generally the more modern terminology.